src/extra/components/pulses.py

```diff
@@ -291,9 +291,6 @@ def get_pulse_mask(self, labelled=True):
     def is_constant_pattern(self):
         """Whether pulse IDs are constant in this data.
 
-        Args:
```

Comment:
I couldn't actually find anything in the [current Google docstring style guide](https://google.github.io/styleguide/pyguide.html) about this, though IIRC this used to be thing. Do you dislike it, or did it cause problems with `mkDocs`?

Reply:
It causes warnings with mkdocs:
```
WARNING  -  griffe: src/extra/components/pulses.py:295: Failed to get 'name: description' pair from 'None'
WARNING  -  griffe: src/extra/components/pulses.py:296: Empty parameters section at line 3
WARNING  -  griffe: src/extra/components/pulses.py:334: Failed to get 'name: description' pair from 'None'
WARNING  -  griffe: src/extra/components/pulses.py:335: Empty parameters section at line 7
WARNING  -  griffe: src/extra/components/pulses.py:374: Failed to get 'name: description' pair from 'None'
WARNING  -  griffe: src/extra/components/pulses.py:375: Empty parameters section at line 3
WARNING  -  griffe: src/extra/components/pulses.py:391: Failed to get 'name: description' pair from 'None'
WARNING  -  griffe: src/extra/components/pulses.py:392: Empty parameters section at line 7
```

Reply:
Ah, fair enough!